packages/web-react/src/components/Toast/demo/ToastDynamicToastQueue.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import { AlignmentXDictionaryType, AlignmentYDictionaryType, ToastColorType } from '../../../types';
+import { Button } from '../../Button';
+import { Checkbox } from '../../Checkbox';
+import { Link } from '../../Link';
+import { Radio } from '../../Radio';
+import { Select } from '../../Select';
+import { Stack } from '../../Stack';
+import { TextArea } from '../../TextArea';
+import { TextField } from '../../TextField';
+import Toast from '../Toast';
+import ToastBar from '../ToastBar';
+import { ToastItem, ToastProvider } from '../ToastContext';
+import { useToast } from '../useToast';
+
+const ToastDynamicToastQueue = () => {
+  const [isCollapsible, setIsCollapsible] = useState(true);
+  const [alignmentY, setAlignmentY] = useState<AlignmentYDictionaryType>('bottom');
+  const [alignmentX, setAlignmentX] = useState<AlignmentXDictionaryType>('center');
+  const [colorValue, setColorValue] = useState<ToastColorType>('inverted');
+  const [hasIconValue, setHasIconValue] = useState(true);
+  const [isDismissibleValue, setIsDismissibleValue] = useState(true);
+  const [messageValue, setMessageValue] = useState('This is a new toast message.');
+
+  const { queue, show, hide, clear, setQueue } = useToast();
```

Comment:
```suggestion
  const { queue, show, hide, clear, setQueue } = useToast(defaultQueue);
```

And the `useEffect` will be useless ;-)

Reply:
will create issue to fix this later